lib/pensio_api/funding_list.rb

```diff
@@ -20,7 +20,7 @@ def initialize(funding_list_body)
       @acquirer = @raw['Acquirer']
       @funding_date = Date.parse(@raw['FundingDate'])
       @created_at = Date.parse(@raw['CreatedDate'])
-      @download_link = @raw['DownloadLink']
+      @download_link = @raw['DownloadLink'].strip
```

Comment:
Some spec examples, when reading the download URL from the XML, were including whitespace, and causing URI to raise an error. This fixes that.